### PR TITLE
Social Links!

### PR DESCRIPTION
**Pull request by nathunsmitty**
Add social links to show pages because yay social media! (Issue #65)

Some pics:
screen shot 2017-02-22 at 6 49 11 pm

Corresponding panel view:
screen shot 2017-02-22 at 6 51 41 pm

Adding some more links: 
screen shot 2017-02-22 at 6 52 23 pm

Seeing that on the site!
screen shot 2017-02-22 at 6 52 28 pm

Some known limitations:
There's no verification of the urls given, what's typed into the panel is what's given as a link, meaning that people could post links that aren't to their social media.

Example:
screen shot 2017-02-22 at 6 57 16 pm

screen shot 2017-02-22 at 6 52 53 pm

Also, no protocol is specified so if a DJ doesn't prepend a http:// or https:// then it'll look for it on uclaradio.com. So hopefully DJs know how to input urls. ¯\_(ツ)_/¯

### CODE REVIEW

File: public/panel/faq.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <html>
   <head>
     <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width; initial-scale=1.0; maximum-scale=1.0; user-scalable=0;">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0">
```
Comment (ChrisLaganiere) [inline public/panel/faq.html]: cool beans

**Comment by ChrisLaganiere** [comment]
Code looks great! I want to get another set of eyes on the design first though ⚠️ 

It's not really clear to me what those icons are. I'm familiar with these sites and it's confusing which one is soundcloud and mixcloud.. can we get a description on those maybe? Like 'Facebook' or even 'episodes' for mixcloud, etc.

**Comment by nathunsmitty** [comment]
Totally! Just for reference, here's [the icon library I'm using](http://fontawesome.io/icons/). Icons used for the social links are:

- [fa-facebook-square](http://fontawesome.io/icon/facebook-square/)
- [fa-tumblr-square](http://fontawesome.io/icon/tumblr-square/)
- [fa-soundcloud](http://fontawesome.io/icon/soundcloud/)
- [fa-mixcloud](http://fontawesome.io/icon/mixcloud/)

I like FontAwesome because it makes my job easy (I only have to add some CSS at the top and use some class names), but I'm open to whatever is clearest and looks best.

**Comment by nathunsmitty** [comment]
Also, I really like the idea of more vibrant icons, but I'm also afraid about the visibility of a lighter color like orange against the background. How would you feel about making the icons display their respective colors on mouse hover? I'll push a commit of what I'm talking about.

**Comment by ChrisLaganiere** [comment]
I like the rollover colors!! Alright, I'm good with this. Any thoughts @nmockovciak ?

**Comment by nmockovciak** [comment]
It looks pretty good! I love the hover colors feature too. 
I think I might want to mess with the overall design of the show pages (and add a way to link to them from the blurbs on the grid view that I'm about to merge on my showTab branch), but that would be sometime soon and the social links were the focus of this branch. So I think you're good to go! Looks nice!

**Comment by nathunsmitty** [comment]
Yeah! I was trying to get this out quickly and was experiencing an issue with "http://" being prepended to empty strings. From talking with @ChrisLaganiere it seemed like we were going to put most of our faith in the djs to enter their urls correctly, so I made the lazy decision to just remove it rather than fully test the feature.

**Comment by nathunsmitty** [comment]
I'm open to re-implementing it if you think that'd be best, though.

**Comment by ChrisLaganiere** [comment]
Hmm, probably a good idea to validate the url to some extent when it's submitted

**Comment by binerys** [comment]
@nathunsmitty @ChrisLaganiere The function I added handles the empty string case and can be extended for additional URL validation. I'll update the branch w/ it so you can check it out and let me know what you guys think!

**Comment by nathunsmitty** [comment]
I'm a fan of this; it's the function I should have written!

**Comment by binerys** [comment]
woot woot, i'd say its ready to go then!

**Pull request merged**